src/parsers/manifestjson.js

```diff
@@ -107,6 +107,13 @@ export default class ManifestJSONParser {
 
     if (error.keyword === 'required') {
       baseObject = messages.MANIFEST_FIELD_REQUIRED;
+    } else if (error.dataPath.startsWith('/permissions') &&
```

Comment:
Is the expectation that you'd get this error for every permission in the list that's not a string?